test/spoof/spoof.go

```diff
@@ -257,6 +259,10 @@ func DefaultErrorRetryChecker(err error) (bool, error) {
 	if isConnectionReset(err) {
 		return true, fmt.Errorf("Retrying for connection reset: %w", err)
 	}
+	// Retry on connection/network errors.
+	if errors.Is(err, io.EOF) {
+		return true, fmt.Errorf("Retrying for: %w", err)
```

Comment:
```suggestion
		return true, fmt.Errorf("retrying for: %w", err)
```

Reply:
The other errors in this function start with a capital. Do you want to change them all? Otherwise I don't think it makes sense to have this single one different.

Reply:
all should be :) we can clean up separately, I can do that, if you don't feel like it. :)

Reply:
Alright. Done in a new commit.